apps/core/scenes/bookingDetail/BookingDetail.js

```diff
@@ -13,7 +16,12 @@ type Props = {|
 
 export default class BookingDetail extends React.Component<Props> {
   renderInner = (props: BookingDetailQueryResponse) => {
-    return <TripDetails data={props.bookingDetail} />;
+    return (
+      <ScrollView contentContainerStyle={styles.container}>
```

Comment:
I think we should move this to a separate component, unless this renderInner will turn into a beast